resources/latest-mac.yml

```diff
@@ -0,0 +1,12 @@
+version: VERSION
+files:
+  - url: Realm Studio-VERSION-mac.zip
+    sha512: ZIP_SHA
+    size: ZIP_SIZE
+    blockMapSize: ZIP_BLOCKMAP_SIZE
+  - url: Realm Studio-VERSION.dmg
+    sha512: DMG_SHA
+    size: DMG_SIZE
+path: Realm Studio-VERSION.zip
+sha512: ZIP_SHA
+RELEASE_DATE
```

Comment:
Was this committed on purpose? If so, what is it used for?

Reply:
This is referenced in the commented block for creating auto-update files for mac in the scenario where electron builder can't be relied on to correctly write that out - not used right now, but I thought it's a good idea to keep that documented if it ever comes up.

Reply:
I'd prefer if this was put into a markdown file with an introduction :+1:

Reply:
Probably best would be sticking it into the comment for that code block, with annotation where it needs to go if that is needed again?